index.js

```diff
@@ -31,7 +37,65 @@ module.exports = {
       // TODO: publish a new group/init message on the group feed
       // TODO: consider what happens if the app crashes between any step
 
-      cb(null, {}) // junk
+      const groupKey = new SecretKey()
+      const content = {
+        type: 'group/init',
+        tangles: {
+          group: { root: null, previous: null },
+        },
+      }
+      //if (!initSpec.isValid(content)) return cb(new Error(initSpec.isValid.errorsString))
+
+      /* enveloping */
+      // we have to do it manually this one time, because the auto-boxing checks for a known groupId
+      // but the groupId is derived from the messageId of this message (which does not exist yet
```

Comment:
I can kind of see what's going on here (would appreciate more context, though!), but using getFeedState as a shortcut to build a ciphertext is bypassing all the work we did with `ssb.db.create`, and building some new technical debt. Let's not

Reply:
I took a look at the modules, and I suggest we update `ssb-box2` so that it can receive, besides strings, also `{key: groupKey.toBuffer(), scheme: keySchemes.private_group}`.

Reply:
We need to access the previous msgId for the feed we're publishing on.
I used this as a quick placeholder to get Jacob underway. Was going to talk with you and read docs to figure out how to get it without compat.

Basically the init message is special and we have to do manual encryption for first message. Ohhhh unless we can put keys in which are { key, scheme } yes...

Reply:
Pending on https://github.com/ssbc/ssb-box2/pull/17 merged first

Reply:
@staltz fixed